### ISSUE TITLE
Add reverse_bimap to Converter class?

### ISSUE THREAD

**Issue by matentzn** [biopragmatics/curies#135]
For various reasons it would be convenient to be able to have a reverse bimap like

```
@property
def reverse_bimap(self) -> Mapping[str, str]:
    """Get the reverse bijective mapping between URI prefixes and CURIE prefixes."""
    return {r.uri_prefix: r.prefix for r in self.records}
```

@cthoyt do you see any good reasons for not having one?

**Comment by matentzn** [comment]
Aweeeesome! Thanks @cthoyt much appreciated!

**Comment by cthoyt** [comment]
release coming soon, but setuptools is globally broken so I have to switch to a different backend

**Comment by cthoyt** [comment]
@matentzn available in curies 0.9.2 - https://pypi.org/project/curies/0.9.2/
